test/index.spec.js

```diff
@@ -190,8 +184,6 @@ describe('InlineSourceWebpackPlugin', () => {
     testInlineSourceWebpackPlugin({
       webpackConfig,
       done,
-      expectOutputFiles: [/index\.html$/, /index\.\w+\.js$/, /runtime\.\w+\.js$/, /bundle\.\w+\.js$/],
-      expectResults: ['<!--inline-source-webpack-plugin-->'],
```

Comment:
When `noAssetMatch` is set to `'error'` webpack 5 doesn't create the `dist/` folder.
I didn't drill too far into why this wasn't built as the hard error indicates a failed build.

Hopefully this isn't a deal-breaker.